python/dask_cudf/dask_cudf/backends.py

```diff
@@ -426,3 +428,66 @@ def sizeof_cudf_dataframe(df):
 @_dask_cudf_nvtx_annotate
 def sizeof_cudf_series_index(obj):
     return obj.memory_usage()
+
+
+try:
+
+    # Define "cudf" backend engine to be registered with Dask
+    from dask.dataframe.backends import DataFrameBackendEntrypoint
+
+    class CudfBackendEntrypoint(DataFrameBackendEntrypoint):
+        @staticmethod
+        def from_dict(data, npartitions, orient="columns", **kwargs):
+            if orient != "columns":
+                raise ValueError(f"orient={orient} is not supported")
+            return dd.from_pandas(
```

Comment:
Is `from_pandas` required here because we don't support `cudf.DataFrame.from_dict` API yet? If so, can we add a todo here to change this after https://github.com/rapidsai/cudf/issues/11934 is resolved?

Reply:
> Is from_pandas required here because we don't support cudf.DataFrame.from_dict API yet?

Yes and no - We should certainly use `cudf.from_dict` when it is supported (I'll add a TODO). However, I'll change the `dd.from_pandas`  code to `dask_cudf.from_cudf` for clarity (`from_cudf` is just a cudf-friendly alias for `from_pandas`).